Modules/Core/Common/include/itkAnatomicalOrientation.h

```diff
@@ -0,0 +1,533 @@
+                               CoordinateEnum::LeftToRight),
+    SPL = ITK_ORIENTATION_ENUM(CoordinateEnum::SuperiorToInferior,
+                               CoordinateEnum::PosteriorToAnterior,
+                               CoordinateEnum::LeftToRight),
+    IAL = ITK_ORIENTATION_ENUM(CoordinateEnum::InferiorToSuperior,
+                               CoordinateEnum::AnteriorToPosterior,
+                               CoordinateEnum::LeftToRight),
+    SAL = ITK_ORIENTATION_ENUM(CoordinateEnum::SuperiorToInferior,
+                               CoordinateEnum::AnteriorToPosterior,
+                               CoordinateEnum::LeftToRight),
+
+    PIR = ITK_ORIENTATION_ENUM(CoordinateEnum::PosteriorToAnterior,
+                               CoordinateEnum::InferiorToSuperior,
+                               CoordinateEnum::RightToLeft),
+    PSR = ITK_ORIENTATION_ENUM(CoordinateEnum::PosteriorToAnterior,
+                               CoordinateEnum::SuperiorToInferior,
+                               CoordinateEnum::RightToLeft),
+    AIR = ITK_ORIENTATION_ENUM(CoordinateEnum::AnteriorToPosterior,
+                               CoordinateEnum::InferiorToSuperior,
+                               CoordinateEnum::RightToLeft),
+    ASR = ITK_ORIENTATION_ENUM(CoordinateEnum::AnteriorToPosterior,
+                               CoordinateEnum::SuperiorToInferior,
+                               CoordinateEnum::RightToLeft),
+    PIL = ITK_ORIENTATION_ENUM(CoordinateEnum::PosteriorToAnterior,
+                               CoordinateEnum::InferiorToSuperior,
+                               CoordinateEnum::LeftToRight),
+    PSL = ITK_ORIENTATION_ENUM(CoordinateEnum::PosteriorToAnterior,
+                               CoordinateEnum::SuperiorToInferior,
+                               CoordinateEnum::LeftToRight),
+    AIL = ITK_ORIENTATION_ENUM(CoordinateEnum::AnteriorToPosterior,
+                               CoordinateEnum::InferiorToSuperior,
+                               CoordinateEnum::LeftToRight),
+    ASL = ITK_ORIENTATION_ENUM(CoordinateEnum::AnteriorToPosterior,
+                               CoordinateEnum::SuperiorToInferior,
+                               CoordinateEnum::LeftToRight)
+
+  };
+
+#undef ITK_ORIENTATION_ENUM
+
+
+  /** \brief Initialize with CoordinateEnum's from separate axes.
+   *
+   * If multiple CoordinateEnums are from the same axes then the Orientation value is INVALID.
+   */
+  AnatomicalOrientation(CoordinateEnum primary, CoordinateEnum secondary, CoordinateEnum tertiary);
+
+  AnatomicalOrientation(ToEnum orientation)
+    : m_Value(orientation)
+  {}
+
+
+#ifndef ITK_FUTURE_LEGACY_REMOVE
+  /** \brief Conversion from Legacy SpatialOrientation
+   *
+   * @param legacyOrientation
+   */
+  AnatomicalOrientation(LegacyOrientationType legacyOrientation)
+#  ifdef ITK_FUTURE_LEGACY_REMOVE
+    [deprecated("Use the AnatomicalOrientation::FromEnum type instead.")]
```

Comment:
This `#ifdef ITK_FUTURE_LEGACY_REMOVE` is unreachable inside `#ifndef ITK_FUTURE_LEGACY_REMOVE` block.

`[[deprecated]]`, not `[deprecated]`

For a function the syntax is e.g. `[[deprecated]] void f();`

Just FYI, with this changes or without -- you know my opinion about all this stuff. I got a notification and looked, I will unsubscribe from the PR to avoid further notifications, no need to respond.

Reply:
Thank you . I have addressed this.